test/src/e2e_vm_tests/test_programs/should_pass/test_contracts/basic_storage/json_storage_slots_oracle.json

```diff
@@ -1,42 +1,46 @@
 [
   {
     "key": "02dac99c283f16bc91b74f6942db7f012699a2ad51272b15207b9cc14a70dbae",
-    "value": "6161610000000000000000000000000000000000000000000000000000000000"
+    "value": "6161000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Are these changes all expected?

Reply:
> Are these changes all expected?

Yes. Because I added another storage field, the storage indices changed, and hence the keys.